cypress/e2e/mod6_homework.cy.js

```diff
@@ -0,0 +1,60 @@
+const url = 'https://jsonplaceholder.typicode.com/posts'
+const url2 = 'https://jsonplaceholder.typicode.com/posts/1'
+
+describe('1st scenario', () => {
+    it('should test placeholder API with POST', function () {
+
+    cy.request(url).then((response) => {
+        const body = JSON.stringify(response.body)
+        expect(response.status).to.eq(200)
+        cy.log(body)
+    })
+
+    cy.request(
+        {
+            method: 'POST',
+            url: url,
+            body: JSON.stringify({
+                title: 'test',
+                body: 'test2',
+                userId: 1
+            }),
+            headers: {'Content-type': 'application/json'}
+        }
+        ).then((response) => {
+            const body = JSON.stringify(response.body)
+            cy.log(body)
```

Comment:
samo wypisanie body to mało - zrób jakieś asercje na to body. Czy to co zwraca jest ok